cfpb/rental-assistance-finder
# Build process and data formatting updates

## Additions
- IE11 support

## Changes

- Update build process to generate a single JS file
- Data updated
- Support processed data

## Testing

1. Check out branch
2. Run yarn start
3. Verify that tool works as expected
4. Run `npm run build` and verify that a single JS file is generated in build folder
